html/partials/exception/broken_desc.html.haml

```diff
@@ -0,0 +1,6 @@
+%h1
+  System Description broken!
+.row
+  .col-xs-1
+  .col-xs-4
+    Broken manifest of system description
```

Comment:
I do not think that this explanation is needed since it is kind of redundant in regard to the real error message.